release_notes.md

```diff
@@ -1,4 +1,5 @@
 ## next (unreleased)
+- [#928](https://github.com/Flank/flank/pull/991) Fail fast on orientation mispell ([sloox](https://github.com/Sloox))
```

Comment:
In [ ] should be the number of the pull request in this case [#991]

Reply:
Thanks 👍